webpack.config.js

```diff
@@ -61,17 +65,23 @@ const externals = {
 	moment: 'moment',
 };
 
-entryPointNames.forEach( entryPointName => {
-	externals[ '@wordpress/' + entryPointName ] = {
-		this: [ 'wp', entryPointName ],
+[ ...entryPointNames, ...packageNames ].forEach( name => {
+	externals[ `@wordpress/${ name }` ] = {
+		this: [ 'wp', name ],
 	};
 } );
 
 const config = {
-	entry: entryPointNames.reduce( ( memo, entryPointName ) => {
-		memo[ entryPointName ] = './' + entryPointName + '/index.js';
-		return memo;
-	}, {} ),
+	entry: Object.assign(
+		entryPointNames.reduce( ( memo, entryPointName ) => {
+			memo[ entryPointName ] = `./${ entryPointName }/index.js`;
+			return memo;
+		}, {} ),
+		packageNames.reduce( ( memo, packageName ) => {
+			memo[ packageName ] = `./node_modules/@wordpress/${ packageName }`;
```

Comment:
Okay, I think I understand the `gitignore` change now, if we're creating a custom build for hooks to expose the `wp.hooks` global. I think there's an issue though, that it's not whitelisted to be included in the plugin zip (see `build-plugin-zip.sh`). I think we have a few of these "list of all top-level folders" scattered throughout the codebase that might need to be audited as well.

Reply:
Yes, I will add it to the whitelist. It seems like this is the only place to update.